src/main.rs

```diff
@@ -14,7 +14,7 @@ use std::time::Instant;
 use time::*;
 
 fn main() {
-    println!(":: Mainframer v2.1.0\n");
+    println!(":: Mainframer v{}\n", env!("CARGO_PKG_VERSION"));
```

Comment:
How does it even work though? When runtime is packaged as a binary it exports environment variables itself?

Reply:
`env!` is a macro that compiler executes at build time, `CARGO_PKG_VERSION` is an environment variable provided by `cargo` to `rustc` during the build, so as a result we get version inlined into the code and binary

Reply:
These native folks will steal your soul eventually, you know. Avoid blood-involved rituals at all costs.

Reply:
lol, will try

btw Kotlin might get macros support eventually